evennia/contrib/test_traits.py

```diff
@@ -0,0 +1,999 @@
+#!/usr/bin/env python
```

Comment:
The position of this file may change. The contrib folder structure itself may see an overhaul. This felt too big to cram into the existing tests.py file.